packages/nvidia_nat_nemo_customizer/tests/test_nemo_customizer.py

```
@@ -0,0 +1,1118 @@
+        trainer_config.continue_on_collection_error = True
+        trainer_config.num_runs = 3
+        trainer = NeMoCustomizerTrainer(trainer_config=trainer_config)
+
+        trajectory_collection = TrajectoryCollection(
+            trajectories=sample_dpo_trajectories,
+            run_id="test_run",
+        )
+
+        call_count = [0]
+
+        async def finalize_side_effect(*args, **kwargs):
```

Ruff (0.14.8)

Unused function argument: `args`

(ARG001)